index.html

```diff
@@ -28,7 +28,7 @@ <h2 class="sd">
                 don’t hestiate to contact me.
             </p>
             <ul class="icon-list">
-                <li><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="https://twitter.com/richardfriday14" target="blank"><img src="./images/twitter.png" alt="twitter"></a></li>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/160498874-4128378d-ca8b-415c-92f7-f90ee149974b.png)

Therefore:

- [ ] Please, kindly use Flexbox to align the elements of your form, as stated in the requirements.

That way, we will meet this particular requirement 💪

Reply:
In the text "
        I'm always interested in hearing about new projects, so if you'd like to
        chat please get in touch.
      "
![image](https://user-images.githubusercontent.com/57421823/160499556-dbaac070-4df3-4aca-bedf-8fa989106a67.png)



- [ ] You have not implemented font styling for this paragraph.  Please add the correct font styling for this `<p>`.  

## Hints

Look at the Figma:

![image](https://user-images.githubusercontent.com/57421823/160499361-4c63e7e4-fc71-4eb3-8450-99545c1bc503.png)

Please, implement these stylings to that `<p>`, so your project looks like the Figma 💪